### DISCUSSION TITLE
What did the axialVelGain,lateralVelGain ,headingVelGain mean?

### DISCUSSION THREAD

**Discussion by 6-BennyLi-9** [acmerobotics/road-runner-quickstart#383] [Q&A]
I am now done most the tuning but these variables: axialVelGain,lateralVelGain ,headingVelGain .

What did they mean?

**Comment by rbrott** [comment]
They are proportional gains on the velocity errors (similar to derivative gains on the position errors). They probably don't need to be touched, though feel free to experiment with them.
